closed/src/java.base/share/native/libjncrypto/NativeCrypto.c

```diff
@@ -38,7 +38,10 @@
 
 #define OPENSSL_VERSION_1_0 "OpenSSL 1.0."
 #define OPENSSL_VERSION_1_1 "OpenSSL 1.1."
-/* needed for OpenSSL 1.0.2 Thread handling routines */
+/* Per new OpenSSL naming convention starting from OpenSSL3, all major versions are ABI and API compatible. */
+#define OPENSSL_VERSION_3_0 "OpenSSL 3."
+
+/* needed for OpenSSL 1.0.2 Thread handling routines. */
```

Comment:
This isn't a sentence; a period is not appropriate.

Reply:
Edited

Reply:
The period is still here.

Reply:
@WilburZjh can you please fix this, I think everything else was addressed.

Reply:
Hi @pshipton , I am not sure whether to remove the period or not, please check this [comment](https://github.com/ibmruntimes/openj9-openjdk-jdk11/pull/499#discussion_r850795604)

Reply:
The other [review comment](https://github.com/ibmruntimes/openj9-openjdk-jdk11/pull/499#discussion_r850795604) seems a different situation, it's code that was incorrect. In this case, Keith has suggested the code comment is not formed as a sentence and so the period at the end should be removed. Either remove the period, or re-write the code comment as a sentence.

Reply:
Edited.

Reply:
You say edited, and I see a force push, but I don't see the expected change.

Reply:
Edited